dbt_semantic_interfaces/implementations/saved_query.py

```diff
@@ -48,3 +52,15 @@ def _implements_protocol(self) -> SavedQuery:
     metadata: Optional[PydanticMetadata] = None
     label: Optional[str] = None
     exports: List[PydanticExport] = Field(default_factory=list)
+    tags: Union[str, List[str]] = Field(
+        default_factory=list,
+    )
+
+    @classmethod
+    def parse_obj(cls, input: Any) -> Self:  # noqa
+        if isinstance(input, dict):
+            if isinstance(input.get("tags"), str):
+                input["tags"] = [input["tags"]]
+            if isinstance(input.get("tags"), list):
+                input["tags"].sort()
```

Comment:
Should `parse_obj` modify the input? We may want to make this more functional

Reply:
Ya, that is what I meant. However, I think that could also be less performant. I'll let you be the judge of what is best.